src/main/kotlin/io/github/yuutoproject/yuutobot/commands/Dialog.kt

```diff
@@ -18,47 +18,147 @@
 
 package io.github.yuutoproject.yuutobot.commands
 
+import com.fasterxml.jackson.core.type.TypeReference
 import io.github.yuutoproject.yuutobot.commands.base.AbstractCommand
 import io.github.yuutoproject.yuutobot.commands.base.CommandCategory
+import io.github.yuutoproject.yuutobot.utils.EMOJI_REGEX
+import io.github.yuutoproject.yuutobot.utils.NONASCII_REGEX
+import io.github.yuutoproject.yuutobot.utils.httpClient
+import io.github.yuutoproject.yuutobot.utils.jackson
 import java.io.IOException
 import net.dv8tion.jda.api.events.message.guild.GuildMessageReceivedEvent
+import net.dv8tion.jda.api.utils.data.DataObject
 import net.dv8tion.jda.internal.utils.IOUtil
-import okhttp3.*
+import okhttp3.Call
 import okhttp3.Callback as OkHttp3Callback
 import okhttp3.MediaType.Companion.toMediaType
+import okhttp3.Request
 import okhttp3.RequestBody.Companion.toRequestBody
+import okhttp3.Response
+import org.slf4j.LoggerFactory
+
+class Dialog : AbstractCommand("dialog", CommandCategory.INFO, "Generates an image of a character in Camp Buddy saying anything you want", "[bg] <char> <text>") {
+    private val logger = LoggerFactory.getLogger(this.javaClass)
+
+    @Suppress("UNCHECKED_CAST")
+    private fun getBackgroundsAndCharacters(): Pair<List<String>, List<String>> {
+        val request = Request.Builder()
+            .url("https://kyuu.to/info")
+            .get()
+            .build()
+
+        httpClient.newCall(request).execute().use { response ->
+            if (response.code != 200) {
+                throw Exception("Failed to sync backgrounds and characters with API")
+            }
+            val json = jackson.readTree(IOUtil.readFully(IOUtil.getBody(response)))
+
+            return Pair(
+                jackson.readValue(json["backgrounds"].traverse(), object : TypeReference<List<String>>() {}),
+                jackson.readValue(json["characters"].traverse(), object : TypeReference<List<String>>() {})
+            )
+        }
+    }
+
+    private val bgAndChars = getBackgroundsAndCharacters()
+    private val backgrounds = bgAndChars.first
+    private val characters = bgAndChars.second
```

Comment:
These are getters meaning that it will probably make an API call every time that they are used, I suggest this
```kotlin
private val backgrounds: List<String> 
private val characters: List<String> 

init {
    val bgAndChars = getBackgroundsAndCharacters()
    
    backgrounds = bgAndChars.first
    characters = bgAndChars.second
}
```